lib/streamlit/elements/markdown.py

```diff
@@ -30,7 +30,7 @@ class MarkdownMixin:
     def markdown(
         self, body: SupportsStr, unsafe_allow_html: bool = False
     ) -> "DeltaGenerator":
-        """Display string formatted as Markdown.
+        r"""Display string formatted as Markdown.
```

Comment:
I guess it's literal and this `r` should not be here, am I right?

Reply:
It should be here. We need to r"""raw triple double quotes""" if we use any backslashes in the docstrings.